File: golang-raw/src/services/users-service.go
package services

import (
	"../models"
	"log"
)

type UsersService struct {
	user  models.UserModel
	users []models.UserModel
}

func (us *UsersService) Init() {
	us.user = models.UserModel{
		Id:        1,
		FirstName: "Hello",
		LastName:  "World",
		Age:       44,
		Hobbies:   []string{"football", "basketball", "guitar", "biking"},
	}
	for i := 0; i < 1000; i++ {
		us.users = append(us.users, models.UserModel{
			Id:        1,
			FirstName: "Hello",
			LastName:  "World",
			Age:       44,
			Hobbies:   []string{"football", "basketball", "guitar", "biking"},
		})
	}
	log.Print("UsersService Init")
}

func (us *UsersService) GetUsers() *[]models.UserModel {
	return &us.users
}

func (us *UsersService) GetUser() *models.UserModel {
	return &us.user
}

File: WebApplication/WebApplication/Services/UsersService.cs
using System.Collections.Generic;
using System.Threading.Tasks;
using WebApplication.DTO;

namespace WebApplication.Services
{
    public class UsersService
    {
        private UserDTO _user;
        private List<UserDTO> _users = new List<UserDTO>();

        public UsersService()
        {
            _user = new UserDTO();
            _user.Id = 1;
            _user.FirstName = "Hello";
            _user.LastName = "World";
            _user.Age = 44;
            _user.Hobbies = new[] {"football", "basketball", "guitar", "biking"};


            for (int i = 0; i < 1000; i++)
            {
                UserDTO userDto = new UserDTO();
                userDto.Id = 1;
                userDto.FirstName = "Hello";
                userDto.LastName = "World";
                userDto.Age = 44;
                userDto.Hobbies = new[] {"football", "basketball", "guitar", "biking"};
                _users.Add(userDto);
            }
        }

        public UserDTO GetUser()
        {
            return _user;
        }

        public List<UserDTO> GetUsers()
        {
            return _users;
        }

        public Task<List<UserDTO>> GetUsersTask()
        {
            return Task.FromResult(_users);
        }
    }
}